src/main/java/TaskManager.java

```diff
@@ -0,0 +1,64 @@
+public class TaskManager {
+    private Task[] tasks;
+    private int numOfTasks;
+
+    public TaskManager(int sizeOfTasks) {
```

Comment:
Perhaps "sizeOfTasks" can be renamed to "capacity" for consistency with Duke.java and to prevent confusion with "sizeOfTask" below